build.gradle

```diff
@@ -17,7 +17,7 @@ plugins {
 }
 
 wrapper {
-    gradleVersion = '6.3'
+    gradleVersion = '6.8'
```

Comment:
@sullis: The wrapper version is defined in our root `build.gradle`.

It must also be updated in a Gradle update PR.

My workflow is:

* Edit the value in `build.gralde`
* Run `./gradlew wrapper`
* Run `./gradlew wrapper` a second time (in case the script generation `gradlew` and `gradlew.bat` was updated with the newer gradle version)